resources/scripts.js

```diff
@@ -104,6 +107,23 @@ function checkGameState() {
     return;
 }
 
+function comeRoll(roll) {
+    for (let i = 0; i < comeBetList.length; i++) {
+        if (roll === 7 || roll === 11) {
+            playerWin('come');
+        }
+        if (roll === 2 || roll === 3 || roll === 12) {
+            playerLose('come');
+        }
+        if (comeBetList[i].point === 1) {
```

Comment:
Why not create a const that holds the `comeBetList[i]` value so you aren't repeating yourself as much? 🤔

Reply:
Will do for secondaryBetList[]